utils/gddrescue/Makefile

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (C) 2018 Lucian Cristian
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gddrescue
+PKG_VERSION:=1.22
```

Comment:
This version is no longer on Debian's mirror. Please update.

Reply:
yes, I've tested with 1.23